src/unit_tests/shared/test_enrollment_op.c

```diff
@@ -374,6 +421,7 @@ void test_verificy_ca_certificate_valid_certificate(void **state) {
 }
 /**********************************************/
 /********** w_enrollment_connect *******/
+
```

Comment:
Code style:
Keep the same new lines logic as in the previous tests:
- No new line after the header.
- 1 new line before the header.

Reply:
Done!